méli-mélo/th-canadaday/canada-day.html

```diff
@@ -186,26 +186,102 @@ <h3>CSS classes</h3>
     <dt><code>.bg-canadaday-beige</code></dt>
     <dd>Set a background colour to beige</dd>
 </dl>
-<h2>Examples</h2>
-<h3>Section example</h3>
+<h3>Examples</h3>
+<h4>Section example</h4>
 <section class="well bg-canadaday-beige">
-    <h2 class="mrgn-tp-0">Example of a heading</h2>
+    <h5 class="h3 mrgn-tp-0">Example of a heading</h5>
     <p>This section is styled with <code>.bg-canadaday-beige</code>.</p>
     <p><a href="#">Link example</a></p>
     <button class="btn-default bg-canadaday-beige" type="button">Button example</button>
 </section>
-<h4>Code</h4>
+<h5>Code</h5>
 <pre><code>&lt;section class="well bg-canadaday-beige"&gt;
-    &lt;h2 class="mrgn-tp-0"&gt;Example of a heading&lt;/h2&gt;
+    &lt;h5 class="h3 mrgn-tp-0"&gt;Example of a heading&lt;/h5&gt;
     &lt;p&gt;This section is styled with &lt;code&gt;.bg-canadaday-beige&lt;/code&gt;.&lt;/p&gt;
     &lt;p&gt;&lt;a href="#"&gt;Link example&lt;/a&gt;&lt;/p&gt;
     &lt;button class="btn-default bg-canadaday-beige" type="button"&gt;Button example&lt;/button&gt;
 &lt;/section&gt;</code></pre>
-<h3>&lt;button&gt; .btn example</h3>
+<h4>&lt;button&gt; .btn example</h4>
 <button class="btn btn-default bg-canadaday-beige" type="button">Button</button>
-<h4>Code</h4>
+<h5>Code</h5>
 <pre><code>&lt;button class="btn btn-default bg-canadaday-beige" type="button"&gt;Button&lt;/button&gt;</code></pre>
-<h3>&lt;a&gt; .btn example</h3>
+<h4>&lt;a&gt; .btn example</h4>
 <a href="#" class="btn btn-default bg-canadaday-beige" role="button">Button</a>
-<h4>Code</h4>
+<h5>Code</h5>
 <pre><code>&lt;a href="#" class="btn btn-default bg-canadaday-beige" role="button"&gt;Button&lt;/a&gt;</code></pre>
+<h2 id="a3">Feature tiles using panels</h2>
+<h3>Appearance</h3>
+<div class="row wb-eqht mrgn-tp-md">
```

Comment:
Suggestion: You can use the equal height CSS plugin which are more performant. 

```suggestion
<div class="row wb-eqht-grd mrgn-tp-md">
```

Reply:
Resolving considering this is only a suggestion and optional.